org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/output/PrintResult.java

```diff
@@ -0,0 +1,67 @@
+/**
+ *
+ */
+package org.metaborg.spoofax.shell.output;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.metaborg.core.source.ISourceRegion;
+
+/**
+ * A result that represents output that is human readable.
+ */
+public class PrintResult implements IResult {
```

Comment:
Result would then be a common ancestor with empty children to let Guice differentiate, or do the binding more explicit. Treating them as separate yields a clearer structure at the cost of minimal (seemingly) redundancy.